src/aria/utils/Json.js

```diff
@@ -835,21 +835,47 @@ var ariaUtilsObject = require("./Object");
              * aria.utils.Json.INVALID_JSON_CONTENT
              * @return {Object}
              */
-            load : function (str, ctxt, errMsg) {
+            load: function (str, ctxt, errMsg) {
+                var JSON = Aria.$global.JSON;
+                if (typeof JSON !== "undefined" && typeof JSON.parse === "function") {
+                    try {
+                        return JSON.parse(str);
+                    } catch (ex) {
+                        // Fallback to eval
+                        this.$logWarn("Cannot parse string with native JSON.parse. Defaulting to eval.\ncontext: %1\nJSON string:\n%2", [ctxt, str]);
```

Comment:
Please move this warning string to `$statics` (just like other error/warning messages).